test/test_runlib.py

```diff
@@ -216,6 +217,75 @@ def test_exclude_subsubdir(self):
     self.assertListEqual(sorted(artifacts_dict.keys()),
         sorted(["foo", "bar", "subdir/foosub1", "subdir/foosub2"]))
 
+class TestInTotoRecordRun(unittest.TestCase):
+  """"Test in_toto_run(name, material_list, product_list,
+  link_cmd_args, key=False, record_byproducts=False). """
```

Comment:
what it feels like is as if you copied the function's signature for reference or something. Do you think that having the parameters in here helps at all?